developers/index.html

```diff
@@ -18,7 +18,7 @@ <h2>Guides</h2>
     <!-- li><a href="http://www.ravenbrook.com/project/p4dti/tool/cgi/bugzilla-schema/"
          >Database Schema Documentation</a></li -->
     <!-- Should point to master API, but its doc doesn't compile due to Perl 5.14 missing in RHEL6. -->
-    <li><a href="../docs/5.0/en/html/api/">The Bugzilla API Documentation</a></li>
+    <li><a href="https://bugzilla.readthedocs.io/en/latest/api/">The Bugzilla API Documentation</a></li>
```

Comment:
https://bugzilla.readthedocs.io/en/5.0/api/index.html seems to work. Maybe it's just the relative link that's the problem?

Reply:
I was reading this page here: https://www.bugzilla.org/developers/index.html